matching-service/config/config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "development": {
+    "dialect": "sqlite",
+    "storage": "./database.sqlite3"
+  },
+  "test": {
+    "dialect": "sqlite",
+    "storage": ":memory"
+  },
+  "production": {
+    "dialect": "sqlite",
```

Comment:
not sure if using sqlite in mem database for prod is a good idea

Reply:
We can change it in a future issue, I just used sqlite in this instance so that I could work on the functionality